app/components/Form/ProjectAttachmentsForm.tsx

```diff
@@ -124,19 +140,25 @@ export function ProjectAttachments({
       )}
       <Table
         paginated
-        totalRowCount={projectRevision.projectAttachmentFormChanges.totalCount}
+        totalRowCount={projectAttachmentFormChanges.totalCount}
         filters={tableFilters}
-        pageQuery={AttachmentsQuery}
```

Comment:
I don't have a page query anymore now that attachments are a component, so I can't use this prop. Not sure if I need to do something to replace it? This is what it was doing:

```/**
   * The top-level query used by the page rendenring the table.
   * If provided, when the table filters, ordering or pagination are updated,
   * the table will reload the query before updating the router,
   * preventing a render with the Suspense fallback while the new data is being fetched.
   */
  pageQuery?: GraphQLTaggedNode;